.coveragerc

```diff
@@ -84,6 +84,9 @@ omit =
     homeassistant/components/azure_devops/const.py
     homeassistant/components/azure_devops/sensor.py
     homeassistant/components/azure_service_bus/*
+    homeassistant/components/barry/const.py
+    homeassistant/components/barry/__init__.py
+    homeassistant/components/barry/sensor.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.
```suggestion
```